nav2_dwb_controller/nav_2d_utils/src/path_ops.cpp

```diff
@@ -52,14 +52,14 @@ nav_2d_msgs::msg::Path2D adjustPlanResolution(
   global_plan_out.poses.push_back(last);
 
   // we can take "holes" in the plan smaller than 2 grid cells (squared = 4)
-  double min_sq_resolution = resolution * resolution * 4.0;
+  double max_sq_resolution = resolution * resolution * 4.0;
```

Comment:
Please change this variable back - its not good form to change code in while developing tests unless you're fixing a legitimate bug. I also think as it was made more sense.